### PR TITLE
[LA][APB-460] Separate subscription request from subscription info

### PR DESCRIPTION
**Pull request by bravesirrobin81**

### CODE REVIEW

File: resources/public/api/conf/0.0/modules/subscriptions.raml [hunk 1]
```diff
@@ -30,6 +30,31 @@ types:
...
+        type: string
+      address:
+        description: The agency's address
+        type: Address
+      utr:
+        description:  The agency's UTR
+        type: string
+
+  SubscriptionInfo:
+    displayName: Subscription Request
```
Comment (frankoid) [inline resources/public/api/conf/0.0/modules/subscriptions.raml]: displayName and description for SubscriptionInfo are the same as for SubscriptionRequest but should presumably be different?

Also the API platform expects JSON Schema not RAML types like we have here (and in agent-client-authorisation). Can merge this and change to JSON Schema later if you like though?

Comment (bravesirrobin81) [inline resources/public/api/conf/0.0/modules/subscriptions.raml]: Yes, they should be. 

Where is this stuff documented? Finding the poor tooling and documentation very frustrating at the moment.

Comment (frankoid) [inline resources/public/api/conf/0.0/modules/subscriptions.raml]: It isn't, yet :( . Ed is working on a RAML producers guide and is going to let me know when it's ready.

Comment (frankoid) [inline resources/public/api/conf/0.0/modules/subscriptions.raml]: I've been pointed at marriage-allowance and self-assessment API as exemplars of RAML that works with the API platform.

Comment (bravesirrobin81) [inline resources/public/api/conf/0.0/modules/subscriptions.raml]: OK, I've fixed the displayName and description. Will hold off on making any further changes until documentation is available. Not willing to chase a moving target.

**Pull request merged**